{
    "$ref": "https://www.cert.ssi.gouv.fr/openapi.json",
    "affected_systems": [
        {
            "description": "CPython sans le dernier correctif de sécurité",
            "product": {
                "name": "CPython",
                "vendor": {
                    "name": "Python",
                    "scada": false
                }
            }
        }
    ],
    "affected_systems_content": "",
    "content": "## Solutions\n\nSe référer au bulletin de sécurité de l'éditeur pour l'obtention des correctifs (cf. section Documentation).",
    "cves": [
        {
            "name": "CVE-2025-4330",
            "url": "https://www.cve.org/CVERecord?id=CVE-2025-4330"
        },
        {
            "name": "CVE-2025-4138",
            "url": "https://www.cve.org/CVERecord?id=CVE-2025-4138"
        },
        {
            "name": "CVE-2025-4517",
            "url": "https://www.cve.org/CVERecord?id=CVE-2025-4517"
        },
        {
            "name": "CVE-2025-4435",
            "url": "https://www.cve.org/CVERecord?id=CVE-2025-4435"
        },
        {
            "name": "CVE-2024-12718",
            "url": "https://www.cve.org/CVERecord?id=CVE-2024-12718"
        }
    ],
    "links": [],
    "reference": "CERTFR-2025-AVI-0475",
    "revisions": [
        {
            "description": "Version initiale",
            "revision_date": "2025-06-04T00:00:00.000000"
        }
    ],
    "risks": [
        {
            "description": "Atteinte à l'intégrité des données"
        },
        {
            "description": "Contournement de la politique de sécurité"
        }
    ],
    "summary": "De multiples vulnérabilités ont été découvertes dans Python. Elles permettent à un attaquant de provoquer une atteinte à l'intégrité des données et un contournement de la politique de sécurité.",
    "title": "Multiples vulnérabilités dans Python",
    "vendor_advisories": [
        {
            "published_at": "2025-06-03",
            "title": "Bulletin de sécurité Python MAXIJJCUUMCL7ATZNDVEGGHUMQMUUKLG",
            "url": "https://mail.python.org/archives/list/security-announce@python.org/thread/MAXIJJCUUMCL7ATZNDVEGGHUMQMUUKLG/"
        }
    ]
}